docs/en/02_creating-new-mfa-methods/01_creating-mfa-method-frontend.md

```diff
@@ -10,9 +10,9 @@ The MFA module provides a clear path for creating additional authentication meth
 implement the front-end portion of the required code, using the Basic Math method as an example. Some prior experience
 with React / Redux is recommended.
 
-The front-end components of MFA make use of [`react-injector`](https://github.com/silverstripe/react-injector/)
+The front-end components of MFA make use of [`lib/Injector`](https://github.com/silverstripe/silverstripe-admin/blob/3/client/src/lib/Injector.js)
 (Injector) to allow sharing of React components and Redux reducers between separate JS bundles. You can find more
-documentation on the Injector API in the [Silverstripe docs](https://docs.silverstripe.org/en/developer_guides/customising_the_admin_interface/reactjs_redux_and_graphql/#the-injector-api).
+documentation on the Injector API in the [Silverstripe docs](https://docs.silverstripe.org/en/developer_guides/customising_the_admin_interface/reactjs_redux_and_graphql/#the-injector-api). This means you'll need to ensure that [silverstripe/admin](https://github.com/silverstripe/silverstripe-admin/) is installed in your project.
```

Comment:
You undid that change - though I think it should still be done as per https://github.com/silverstripe/silverstripe-mfa/pull/584#discussion_r1919164519

Either this change needs to be reverted and included in follow-up work, or we should replace react-injector with the admin one as discussed in that linked comment.

Reply:
Reverted